pkg/network/network.go

```diff
@@ -94,10 +124,31 @@ func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
 	} else {
 		nc.IstioOutboundIPRanges = normalizedIpr
 	}
+
 	if ingressClass, ok := configMap.Data[DefaultClusterIngressClassKey]; !ok {
 		nc.DefaultClusterIngressClass = IstioIngressClassName
 	} else {
 		nc.DefaultClusterIngressClass = ingressClass
 	}
+
+	// If not in the config data then it'll be empty, which is ok, we'll
+	// just use the default value at that point
+	nc.RouteTemplateText = configMap.Data[RouteTemplateKey]
+
+	text := nc.RouteTemplateText
+	if text == "" {
+		// Blank makes no sense so use our default instead
+		text = DefaultRouteTemplateText
+	}
```

Comment:
If we really want the text in the config, should the Default make it in there as well? (Currently it doesn't)

Reply:
if we have a reason for adding it I can, but I can't think of any use for it outside of this func.